shared/fs/footer/downloads-container.tsx

```diff
@@ -24,15 +23,10 @@ const mergeProps = ({_downloads}, {openDownloadFolder}) => {
       return b.state.startedAt - a.state.startedAt // newer first
     })
     .map(([key, download]) => key)
-  return ({
+  return {
     downloadKeys,
     openDownloadFolder,
-  }: DownloadsProps)
+  } as DownloadsProps
```

Comment:
Why do we need this `as`? Can you give the return type of `mergeProps` in its declaration instead?

I admit I don't fully understand TS's type inference but I found it useful in once place to fill in the template parameters to `namedConnect`, maybe that would help avoid this?

Reply:
I think this is just a direct translation from the old code. I'll update it to have it in the declaration.